goil/templates/config/rh850/config.oil

```diff
@@ -23,6 +23,10 @@ IMPLEMENTATION renesas {
   OS {
     BOOLEAN [
       TRUE {
+        ENUM [
+          serial
```

Comment:
The CAN library has been declared here https://github.com/TrampolineRTOS/trampoline/blob/7cd2d87808a3b2b8d35c182855eb3b17793fae7e/goil/templates/config/rh850/g4mh/config.oil#L14 to be specific to the G4MH core, in case other RH850 cores use different IP blocks. Do you want to keep the UART here or to move to the G4MH directory ?

Reply:
You're right: g4mh is a better place for it. I'll update that.